test/unit/PointViewTest.cpp

```diff
@@ -343,6 +343,17 @@ TEST(PointViewTest, issue1264)
     EXPECT_THROW(v.setField(foo, 0, d), pdal_error);
 }
 
+TEST(PointViewTest, getFloatNan)
+{
+    PointTable table;
+    PointLayoutPtr layout(table.layout());
+    layout->registerDim(Dimension::Id::ScanAngleRank, Dimension::Type::Float);
+    PointViewPtr view(new PointView(table));
+    const float scanAngleRank = std::numeric_limits<float>::quiet_NaN();
+    view->setField(Dimension::Id::ScanAngleRank, 0, scanAngleRank);
+    float retrieved = view->getFieldAs<float>(Dimension::Id::ScanAngleRank, 0);
```

Comment:
Is this testing that the last line doesn't throw?  If so, `EXPECT_NO_THROW` would make it more explicit.

Reply:
An oversight.  Thanks.